scripts/env.sh

```diff
@@ -16,3 +17,9 @@ export APP_ARTIFACT_ID=${APP_ARTIFACT_ID:-flink-tools}
 export APP_VERSION=${APP_VERSION:-0.2.0}
 export GRADLE_OPTIONS="${GRADLE_OPTIONS:-"-PincludeHadoopS3=false -Pversion=${APP_VERSION}"}"
 export FLINK_IMAGE_TAG="1.10.0-2.12-1.2-W2-4-0577915d2"
+export NEW_IMAGE_TAG="${FLINK_IMAGE_TAG}-hadoop2.8.3"
+export DOCKER_IMAGE_TAR=${ROOT_DIR}/build/flink-${NEW_IMAGE_TAG}.tar
+export SDP_INSTALL_PATH=$HOME/nautilus-install/
```

Comment:
I think that the latest procedure is to put the installer somewhere in ~/desdp. Check with Kartik.

Reply:
At present wiki page has same path $HOME/nautilus-instal. May be we can change once Kartik decide to change to ~/desdp